src/csharp/Microsoft.Spark.UnitTest/Sql/ColumnTests.cs

```diff
@@ -363,6 +363,115 @@ public void TestOver()
             }
         }
 
+        [Fact]
+        public void TestIsIn(){
```

Comment:
Can you please follow the existing coding style?

Reply:
i have added the indentation and the newline before the brace